src/styles/blocks/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 16px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image-wrap {
+    margin-top: 16px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    object-fit: cover;
+  }
+
+  &__info {
+    margin-top: 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $accent-text-color;
+
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-text-color;
+
+    margin-bottom: 0;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    width: 100%;
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: end;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $accent-text-color;
+  }
+
+  &__price-tag {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price {
+    color: $secondary-text-color;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
```

Use scss nesting everywhere
```suggestion
  &__price {
    color: $secondary-text-color;
    font-size: 12px;
    font-weight: 400;
    line-height: 18px;
    
    &-value {
      // write styles here
    }
  }
```